### PR TITLE
PR: Update docs

### LINKED ISSUE #12: Release spyder-env-manager 0.1.0
* Update requirement to envs-manager >= 0.1.3
* Update requirement to spyder>= 6.0.0alpha1 and update CI to test using Spyder master branch
* Add some tests
* Update README, docs and add RELEASE
* Do initial UX/UI fixes round

Depends on the release of the first Spyder 6 alpha: https://github.com/spyder-ide/spyder/milestone/119

### PR DESCRIPTION
**Pull request by dalthviz**
Part of #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ Spyder 5+ plugin to manage Python virtual environments and packages
 
 ## Features
 
-* TODO
+* Create and delete Python environments
+* Install, Update, change installed packages inside Python environments
+* Import/Export environments (from/to `.yml` files)
+* Set the environment Python interpreter as the one Spyder uses
```
Comment (ccordoba12) [inline README.md]:
```suggestion
* Set the environment Python interpreter as the one used by Spyder.
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+* Clean your local repo with (selecting option 1)
+
+      git clean -xfdi
+
+* Update `CHANGELOG.md` with
+
+      loghub spyder-ide/spyder-env-manager -m vX.Y.Z
+
+* Update `__version__` in `spyder/__init__.py` (set release version, remove `dev0`):
```
Comment (ccordoba12) [inline RELEASE.md]:
```suggestion
* Update `__version__` in `spyder_env_manager/__init__.py` (set release version, remove `dev0`):
```

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+* Upload generated files with
+
+      twine upload dist/*
+
+* Create release tag with
+
+      git tag -a vX.Y.Z -m "Release X.Y.Z"
+
+* Update `__version__` in `spyder/__init__.py` (add `dev0` and increment minor)
```
Comment (ccordoba12) [inline RELEASE.md]:
```suggestion
* Update `__version__` in `spyder_env_manager/__init__.py` (add `dev0` and increment minor)
```

**Review by ccordoba12 — COMMENTED** [review]
Some small suggestions for you @dalthviz, otherwise looks good to me.

**Comment by ccordoba12** [comment]
I think before releasing the first version of this package, we need to publish the first alpha of Spyder 6, right?

**Comment by dalthviz** [comment]
> I think before releasing the first version of this package, we need to publish the first alpha of Spyder 6, right?

Probably, I'm not sure what could be the best approach 🤔

**Comment by ccordoba12** [comment]
I think that would be the best we can do because if you publish an initial version of this package right now, pip/conda won't be able to resolve its dependency on Spyder 6.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Pull request merged**